libs/partners/pinecone/pyproject.toml

```
@@ -14,7 +14,7 @@ license = "MIT"
 # <3.13 is due to restriction in pinecone-client package
 python = ">=3.8.1,<3.13"
 langchain-core = ">=0.1.52,<0.3"
-pinecone-client = "^3.2.2"
+pinecone-client = ">=3.2.2"
```

```suggestion
pinecone-client = ">=3.2.2,<5"
```
should we cap it at 5 in the event that major release is a breaking change?